hledger: add page (#8635)
tldr-pages/tldr: pages/common/hledger.md

# hledger

> > A plain text accounting software for the command line.

Reviewer: ```suggestion
> A plain text accounting software for the command-line.
```

> More information: <https://hledger.org>.

> - Use the interactive `add` command to add transactions to your journal:

Reviewer: Perhaps this better
```suggestion
- Add transactions to your journal interactively:
```

`hledger add`

> - Show the account hierarchy, using a journal file in a non-standard location:

Reviewer: Perhaps this better
```suggestion
- Show the account hierarchy, using a specific journal file:
```
